frontend/src/store/authStore.ts

```diff
@@ -16,10 +16,19 @@ export const useAuthStore = defineStore('auth', () => {
 
   // Getters
   const getKeycloak = computed(() => _keycloak.value);
-  // const getAuthenticated = computed((): boolean => (ready.value ? !!_keycloak.value.authenticated : false));
-  // const getLoginUrl = computed(() => (ready.value ? _keycloak.value.createLoginUrl() : ''));
-  // const getToken = computed(() => (ready.value ? _keycloak.value.token : ''));
-  // const getTokenParsed = computed((): Object | undefined => (ready.value ? _keycloak.value.tokenParsed : {}));
+  const getIdentityProvider = computed(() => _keycloak.value.tokenParsed?.identity_provider);
+
+  function getIdentityId() {
+    if (getIdentityProvider.value === IDENTITY_PROVIDERS.IDIR) {
+      return _keycloak.value.tokenParsed?.idir_user_guid;
+    }
+    // TODO: Confirm BCEID is correct
+    if (getIdentityProvider.value === IDENTITY_PROVIDERS.BCEID_BASIC) {
+      return _keycloak.value.tokenParsed?.bceid_user_guid;
+    }
+
```

Comment:
I beleieve we are allowing bceid business users to log in so we need another item:
You can add
```js
if (getIdentityProvider.value === IDENTITY_PROVIDERS.BCEID_BUSINESS) {
      return _keycloak.value.tokenParsed?.bceid_user_guid;
    }
```
The field from the token we will use for identityId is also `bceid_user_guid`.